cluster/aws/util.sh

```diff
@@ -920,6 +920,11 @@ function start-flannel-ds {
   popd
 }
 
+function start-weave-ds {
+  local kubectl_ver="$(${KUBE_ROOT}/cluster/kubectl.sh version | base64 | tr -d '\n')"
```

Comment:
why is weave network and pod-network-cidr stuff related to your changes? If it's for something else please move them to a separate PR. thanks.

Reply:
Moved to PR https://github.com/futurewei-cloud/arktos/pull/235